mods/ctl/src/commands/providers/create.ts

```diff
@@ -67,23 +67,6 @@ export default class CreateCommand extends Command {
         choices: ["tcp", "udp"],
         default: "tcp"
       },
-      {
```

Comment:
We must keep the prompts in the CTL and the backend will decide whether or not to ignore it.